backend/protzilla/methods/importing.py

```diff
@@ -114,6 +137,35 @@ class MsFraggerImport(ImportingStep):
 
     output_keys = ["protein_df"]
 
+    def create_form(self):
+        return Form(
+            label="DIA-NN Import",
+            input_fields=[
+                FileInput(
+                    name="file_path",
+                    label="MSFragger intensities file (combined_proteins.tsv)",
+                    value=None,
+                ),
+                DropdownField(
+                    name="intensity_name",
+                    label="intensity name",
+                    value=IntensityNameType.INTENSITY, #just a guess
+                    options=IntensityNameType,
+                ),
+                CheckboxField(
+                    name = "map_to_uniprot",
+                    label = "Map to Uniprot IDs using Biomart (online)",
+                    value = False
+                ),
+                DropdownField(
+                    name = "aggregation_method",
+                    label = "Aggregation method used to aggregate duplicate values for protein groups",
+                    value = AggregationMethods.sum,
```

Comment:
```suggestion
                    value = AggregationMethods.sum.value,
```
Sollte hier so verwendet werden, weil das an anderen stellen notwendig ist und so einheitlicher werden kann